client/js/app/utils/ProjectUtils.js

```diff
@@ -139,18 +239,27 @@ module.exports = {
     return collection ? collection.properties[propertyName] : null;
   },
 
+  /*
+    Returns the local timezone offset in seconds offset from UTC.
+    This is how the Keen API wants the offset to look. This is also
+    opposite in negative/positive numbers from how Javascript
+    handles it.
+   */
   getLocalTimezoneOffset: function(date){
-    var offset = new Date().getTimezoneOffset();
-    if (DateUtils.isDST()) {
-      offset += 60;
-    }
-    var strSign = offset > 0 ? '-' : '+';
-    var strHours = FormatUtils.padLeft(Math.floor(offset / 60));
-    var strMinutes = FormatUtils.padLeft(offset % 60);
-    var found = _.find(CONSTANTS.TIMEZONES, function(timezone){
-      return timezone.offset === strSign + strHours + ':' + strMinutes;
+    return (new Date().getTimezoneOffset() * -1) * 60;
+  },
+
+  getLocalTimezoneName: function(date){
+    var isDST = DateUtils.isDST();
+    var localOffset = module.exports.getLocalTimezoneOffset();
+    var zones = CONSTANTS.TIMEZONES.filter((zone) => {
+      if (isDST) {
+        return zone.dst_offset === localOffset;
+      } else {
+        return zone.offset === localOffset;
+      }
     });
-    return found ? found.value : offset * -60;
```

Comment:
The previous `offset` value here would sub in the numeric offset for the locale when a named timezone wasn't present.. would this behavior still be present in this new update?